deployments/helm/templates/configmap.yaml

```diff
@@ -15,5 +15,17 @@ metadata:
   {{- end }}
 data:
   beyla-config.yml: |
+    {{- if eq .Values.preset "network" }}
+    {{- if not .Values.config.data.network }}
+    network:
+      enable: true
+    {{- end }}
+    {{- else }}
```

Comment:
I think we should maybe leave out the else and do another if for .Values.preset "application"? I think we can use this approach to enable selectively application_span metrics.